### PR TITLE
[tools/vpdbootmanager] Added command `get`

### PR DESCRIPTION
**Pull request by insomniacslk**
The `get` command in vpdbootmanager allows to read VPD variables.
Example:
```
~/# vpdbootmanager get firmware_version
firmware_version(RO) => 0.0.0
```

If no arguments are passed to `get`, all the variables are read:
```
~/# vpdbootmanager get
firmware_version(RO) => 0.0.0
internal_versions(RO) => {
  "build_id": "6bc0645",
...
    }
  }
}
~/#
```

Signed-off-by: Andrea Barberio <insomniac@slackware.it>

### CODE REVIEW

**Comment by insomniacslk** [comment]
CC @johnnylinwiwynn

**Comment by insomniacslk** [comment]
Some details on this change:
1. I added a new "get" subcommand to `tools/vpdbootmanager`
2. also added tests, which highlighted the need to refactor `pkg/vpd` to better test it and stop using a global variable for the base VPD path. Now it is a Reader structure with a default global instance

Next steps (in a separate PR):
* change the default behaviour of `get` to only print BootXXXX variables, and add a new flag that would also show the non-boot variables
* fix the file inclusion bug in the use of `path.Join` in `pkg/vpd`

File: tools/vpdbootmanager/main.go [hunk 1]
```diff
@@ -30,23 +34,31 @@ Global flags:
...
 	}
 	switch args[0] {
 	case "add":
 		return add(args[1], args[2:])
+	case "get":
+		varname := ""
+		if len(args) > 1 {
+			varname = args[1]
+		}
+		getter := Getter{}
```
Comment (johnnylinwiwynn) [inline tools/vpdbootmanager/main.go:60]: I think it should be
getter := NewGetter()

Comment (insomniacslk) [reply tools/vpdbootmanager/main.go:60]: indeed it does! I forgot to use it after I added `NewGetter`, good catch

Comment (insomniacslk) [reply tools/vpdbootmanager/main.go]: done

**Review by rminnich — APPROVED** [review]

**Review by rminnich — APPROVED** [review]

**Pull request merged**